templates/vue/src/components/Tapestry.vue

```diff
@@ -215,7 +235,10 @@ export default {
         group: 1,
         typeData: {
           linkMetadata: null,
-          progress: [{ group: "viewed", value: 0 }, { group: "unviewed", value: 1 }],
+          progress: [
+            { group: "viewed", value: 0 },
+            { group: "unviewed", value: 1 },
+          ],
```

Comment:
If you're doing this here, make sure you also make the same change on master to minimize conflicts.